container.nix

```
@@ -310,7 +310,7 @@ let
     };
 
     notify = quadletUtils.mkOption {
-      type = types.nullOr types.bool;
+      type = types.nullOr (types.enum [ "true" "false" "healthy" ]);
```

for backwards compatibility, let's:

```suggestion
      type = types.enum [ null true false "healthy" ];
```

---

I didn't realize Enum can include different types. Nice!